### ISSUE TITLE
Password resharing status/display

### ISSUE THREAD

**Issue by Siemir** [marius-wieschollek/passwords#399]
**Current Status**
Currently when you share password with someone and this person reshares it with someone else, the original owner does not know about it. After this it is possible to share the same password from the owner account and the recipient will have two differents entries which are exactly the same.

**Feature Description**
When password is reshared it should be visible to the owner (and probably everyone, but without any share/edit options - same as in file shares in Nextcloud).

**Additional context**
Also currently it is possible to "break" non editable passwords - if a person without edit permissions shares password to someone, it can be shared with edit permissions, after that said person can edit password without a problem, but the change is visible only to the person who shared password, orginal owner sees non edited password.

**Comment by marius-wieschollek** [comment]
I can reproduce the issue with sharing passwords as editable. I opened the bug https://github.com/marius-wieschollek/passwords/issues/400 for this.

**Comment by Siemir** [comment]
I can see the problem with editing has been solved.

What about presenting all users that have access to password - can this functionality be added?

**Comment by heeplr** [comment]
> When password is reshared it should be visible to the owner 

I just found out that this is not the case an was very surprised. With sensible info like login credentials, you naturally want to know who has access. If someone is re-sharing with the wrong person (by accident), there is no way for the owner to find out.

I can't see any use-case for the owner to not see all existing (re)shares and thus would consider this being a bug.

**Comment by pieter-groeneweg** [comment]
I very much like the feature of sharing passwords. 

But imho the 'owner' of the password should be able to prevent re-sharing and editing.
Also it would be nice if the list of passwords would show the shares just like in 'files'.

**Comment by marius-wieschollek** [comment]
The owner is able to prevent resharing and editing:
![image](https://user-images.githubusercontent.com/8255857/211776345-691a7533-f4e2-4a34-8eb7-1fa5a69dbd7e.png)

**Comment by pieter-groeneweg** [comment]
Ok great thanks, this was not obvious for me... 

I see there is an options under "more" where these can be set (enabled by default?)
These settings are available to all users. Is there a way to set the defaults for this to disabled?

**Comment by marius-wieschollek** [comment]
Yes, these settings follow the global settings for Nextcloud. So you can set the defaults there.

**Comment by pieter-groeneweg** [comment]
Sorry, but I cannot find these in the global settings...

administration settings -> administration: passwords doesn't show these two options.

**Comment by marius-wieschollek** [comment]
As i said, these are the global settings for Nextcloud, so they can be found under "Sharing" in the admin settings

**Comment by pieter-groeneweg** [comment]
A yes, I so these are treated equally to files and folders. 
Thanks, I wasn't aware of that.
Thank you for your help, much appreciated!
